### ISSUE TITLE
Move files and folders on remotes using drag and drop

### ISSUE THREAD

**Issue by dia3olik** [kapitainsky/RcloneBrowser#28]
At least on Mac it's working only if you bring something from the finder (ie local) with drag and drop to the desired directory on the remote files list GUI which brings up the upload window.

But it would be very useful to being able to also quickly move around files and folders already on the cloud (from one directory to another) using this GUI and drag and drop function with an option in the preferences to disable the dialog window to appear every time you move something to another directory to make even faster and more similar to how things work locally! ;-)

**Comment by dia3olik** [comment]
Just to describe it further...it's just that there isn't a quick and intuitive way to move around files already on the remote cloud storage from inside the GUI...creating jobs for this simple task (cloud drive to same cloud drive move) is not a clean approach I think...no?? ;-)

**Comment by kapitainsky** [comment]
Yes I see what you mean. No plans for this though yet. Drag and drop in QT is not something I am familiar with.

It was low hanging fruit for uploads but to drag item from within QT interface required a lot of coding. Definitely doable but not easy.

Let's keep it in mind for future enhancements.

**Comment by caspertone2003** [comment]
>  It was low hanging fruit for uploads but to drag item from within QT interface requires a lot of coding. Definitely doable but not easy.
> 
Not used to it I discovered it today.
You are transforming what used to be a zombie in a full flesh superbaby!
Thanks!
CT
